composer.json

```diff
@@ -22,14 +22,17 @@
     "require-dev": {
         "pear/archive_tar": "^1.4.3",
         "phpunit/phpunit": "^5.7.23 || ^6.4.3",
+        "psr/http-message": "^1.0",
```

Comment:
This package is now strictly required for the class `Zend\Filter\File\RenameUpload`. I think that will result in a BC break if we do not move it to the require section.

Reply:
My thinking was that if PSR payload was given, then they already must have this dependency and implementation somewhere else to generate it (eg. use in expressive that already has it).
The ones who do not and stay on HTTP $_FILES should not have unused dependencies installed: PSR path will not execute and `instanceof` operator works without those classes needing to be autoloaded.

Is that valid reasoning?